sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
+    if not array or len(array) <= 1:
+        return array # nothing to sort
+
+    index = 0
+    swapped = True
+    while index < len(array) and swapped:
+        swapped = False
```

Comment:
is there a ready why we are initially setting "swapped" to 'True', if the first action we take is to switch it to 'False'?  This might not matter but did catch my eye.